invenio_deposit/storage.py

```diff
@@ -42,10 +42,8 @@ class ExternalFile(object):
 
     def __init__(self, url, filename):
         """Initialiez external file."""
-        from invenio.legacy.bibdocfile.api import open_url, \
-            InvenioBibdocfileUnauthorizedURL
         try:
-            self._file = open_url(url, headers={})
+            self._file = urllib2.urlopen(urllib2.Request(url))
```

Comment:
The `open_url` was also adding Invenio special `user-agent` header. Can this be added here then? See: https://github.com/inveniosoftware/invenio/blob/master/invenio/legacy/bibdocfile/api.py#L4890